source/whats-new.txt

```diff
@@ -0,0 +1,318 @@
+.. _c-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade
+
+Learn what's new in:
+
+* :ref:`Version 1.27.6 <c-version-1.27.6>`
+* :ref:`Version 1.27.5 <c-version-1.27.5>`
+* :ref:`Version 1.27.4 <c-version-1.27.4>`
+* :ref:`Version 1.27.3 <c-version-1.27.3>`
+* :ref:`Version 1.27.2 <c-version-1.27.2>`
+* :ref:`Version 1.27.1 <c-version-1.27.1>`
+* :ref:`Version 1.27.0 <c-version-1.27.0>`
+* :ref:`Version 1.26.2 <c-version-1.26.2>`
+* :ref:`Version 1.26.1 <c-version-1.26.1>`
+* :ref:`Version 1.26.0 <c-version-1.26.0>`
+
+.. _c-upcoming-breaking-changes:
+
+.. _c-version-1.27.6:
+
+What's New in 1.27.6
+--------------------
+
+The 1.27.6 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.6 release of ``libbson`` includes the following changes:
+
+- Fixes handling of malformed extended JSON for special BSON types.
+
+libmongoc
+~~~~~~~~~
+
+The 1.27.6 release of ``libmongoc`` includes the following changes:
+
+Fixes
+`````
+
+- Fixes TSan warning.
+- Fixes C23 compile.
+
+Improvements
+```````````
+
+- Documents expected behavior of command errors in a transaction.
+
+.. _c-version-1.27.5:
+
+What's New in 1.27.5
+--------------------
+
+The 1.27.5 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.5 release of ``libbson`` includes the following changes:
+
+- Fixes large string handling in ``bson_append_utf8``.
```

Comment:
S: remove the period, since this isn't a complete sentence.
```suggestion
- Fixes large string handling in ``bson_append_utf8``
```

Reply:
Applies to all (except bullet points that have two sentences, like lines 257-259).